src/utils/BatchManager.js

```diff
@@ -1,5 +1,8 @@
-function errorToSerializable(err) {
-  if (!err) return null;
+function errorToSerializable(error) {
+  if (error === undefined) return null;
```

Comment:
i think that if it's called with no error, then it was called with `undefined`, and i think that's still an error.

for now LGTM but yeah let's only call this function when we have an error